test/Aqua.jl

```diff
@@ -6,6 +6,7 @@ using Aqua
     ambiguities=false,      # TODO: fix ambiguities
     unbound_args=false,     # TODO: fix unbound type parameters
     piracies=false,         # TODO: check the reported methods to be moved upstream
+    persistent_tasks=false  # TODO: FIXME | setting this to true breaks Oscar.build()
```

Comment:
```suggestion
```
should not be an issue anymore due to `subset="short"`

Reply:
Do you still want the error message from when it fails, or do we ignore that since we don't run into it if we only run short tests ?

Reply:
I have seen it myself now. This could be due to https://github.com/JuliaTesting/Aqua.jl/issues/292. But as commented here https://github.com/oscar-system/Oscar.jl/pull/3728#issuecomment-2112279986, I don't think you should be running the Aqua tests at all